airflow/dags/decia_2010_territory_population.py

```diff
@@ -0,0 +1,63 @@
+from airflow import DAG
+from airflow.models import Variable  # type: ignore
+from airflow.utils.dates import days_ago  # type: ignore
+
+import util
+
+_DECIA_2010_POPULATION_GCS_FILENAMES = (
+    'decia_2010_territory_population-by_race_and_ethnicity_territory.json,'
+    'decia_2010_territory_population-by_sex_territory.json,'
```

Comment:
The commas may be in the wrong place here. 

```
_DECIA_2010_POPULATION_GCS_FILENAMES = (
    'decia_2010_territory_population-by_race_and_ethnicity_territory.json',
    'decia_2010_territory_population-by_sex_territory.json',
    'decia_2010_territory_population-by_age_territory.json'
)
```

Reply:
oh weird, ya before they were inside the strings.... why? @joshzarrabi ?

code before:
```
_ACS_2010_POPULATION_GCS_FILENAMES = (
    'acs_2010_population-by_race_and_ethnicity_territory.json,'
    'acs_2010_population-by_sex_territory.json,'
    'acs_2010_population-by_age_territory.json'
)
```

Reply:
good catch. for future reference, this is just making one long, comma separated string, in the strange way python decided to implement multi-line strings